src/main/resources/application.conf

```diff
@@ -2,5 +2,8 @@ application {
   save_path = "twitter-analytics/dataLake/eng-tweets"
   filter = "en"
   hashtags = "trump"
+}
+reporting {
   reportSavePath = "twitter-analytics/reporting"
+  eventDateOfDataForReport = "2018-11-02"
```

Comment:
so, do we need to package/build different artifact for each date?

Reply:
Fixed. This particular property which is more dynamic in nature is moved to an external config file that is external to jar.